packages/angular/migrations.json

```diff
@@ -1303,6 +1303,10 @@
         "ng-packagr": {
           "version": "~16.2.0",
           "alwaysAddToPackageJson": false
+        },
+        "esbuild": {
+          "version": "^0.19.2",
+          "alwaysAddToPackageJson": false
         }
```

Comment:
This should go in a separate package update targeting a future version. Those who are already in 16.7.0 wouldn't get this package update.

Reply:
oh yes, good catch, this PR had been created before the release of 16.7.0